### PR TITLE
[MinGW + MSYS2] update documentation and setup

### PR DESCRIPTION
**Pull request by Amphaal**
...and since LLVM 10 is now live in MSYS2, maybe re-enable MinGW CI builds ? It may require https://github.com/getsentry/crashpad/pull/13 to pass first tho.

### CODE REVIEW

**Comment by Swatinem** [comment]
> maybe re-enable MinGW CI builds ?

We can give it a shot at least. I’m a bit hesitant though, since I don’t want to spend too much time maintaining it myself.

**Comment by Amphaal** [comment]
@Swatinem I just noted that my Remark VSCode extension auto-formated the README.md. Is this an issue ?

**Comment by Swatinem** [comment]
> I just noted that my Remark VSCode extension auto-formated the README.md. Is this an issue ?

It would be nice to revert that, yes. I have autoformatting for markdown in vscode, I think it comes from prettier, but not sure.

**Comment by Amphaal** [comment]
Alright, I think this closes #238 !

File: README.md [hunk 1]
```diff
@@ -115,6 +115,26 @@ or use it on the command line.
 [cmake]: https://cmake.org/cmake/help/latest/
 [cmake guide]: https://developer.android.com/ndk/guides/cmake
 
+**MinGW**:
+
+64-bits is the only plateform supported for now.
```
Comment (Mixaill) [inline README.md:120]: typo: pla**tf**orm

Comment (Amphaal) [reply README.md]: Unleashing my inner french !

**Pull request merged**